src/huts/create/endpointSpec.js

```diff
@@ -19,7 +19,7 @@ describe("'create' endpoint for huts" , function() {
         hutData = {
             hutName: "a hutname for testing puposes",
             street: "test street",
-            streetNumber: "1",
+            streetNumber: "1", 
```

Comment:
Dangly space why

Reply:
If you make sure to run `git diff` before you add files, or `git diff --cached` before you commit, you can avoid committing little odd things like this :smile:

Reply:
oh damn, sorry 😅